packages/ui/src/components/Input/textArea.tsx

```diff
@@ -0,0 +1,156 @@
+import { Input as DefaultInput } from 'antd'
+import { TextAreaProps as AntdTextAreaProps } from 'antd/es/input'
+import classNames from 'classnames'
+import React, { forwardRef, useState, useEffect, TextareaHTMLAttributes } from 'react'
+
+import { ArrowUp } from '@open-condo/icons'
+
+import { Button } from '../Button'
+
+import type { InputRef } from 'antd'
+
+const { TextArea: DefaultTextArea } = DefaultInput
+
+export const TEXTAREA_CLASS_PREFIX = 'condo-input'
+
+export type TextAreaProps = Omit<TextareaHTMLAttributes<HTMLTextAreaElement>, 'style' | 'size' | 'onResize'> &
+Pick<AntdTextAreaProps, 'autoSize'> & {
+    value?: string
+    isSubmitDisabled?: boolean
+    showCount?: boolean
+    onSubmit?: (value: string) => void
+    bottomPanelUtils?: React.ReactElement[]
+}
+
+const TextArea = forwardRef<InputRef, TextAreaProps>((props, ref) => {
+    const {
+        className,
+        disabled,
+        onSubmit,
+        autoFocus,
+        maxLength = 1000,
+        showCount = true,
+        isSubmitDisabled,
+        value: propsValue,
+        bottomPanelUtils = [],
+        onChange: propsOnChange,
+        autoSize = { minRows: 1 },
+        ...restProps
+    } = props
+
+    const [internalValue, setInternalValue] = useState('')
```

Comment:
`internalValue` must be `value`, not empty string
```suggestion
    const [internalValue, setInternalValue] = useState(value)
```